homework3.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"unicode/utf8"
+)
+
+/*Task 1 : average([...]int) float64 function that returns
+an average value of array (sum / N)*/
+func average(array [6]int) float64 {
```

Comment:
Why `[6]int` ? `array` should have `[...]int` type (according to the task description.